.github/workflows/cbindgen.yml

```
@@ -0,0 +1,36 @@
+name: Check cbindgen output
+
+on:
```

The `ci.yml` uses the following. I think we should do the same here.

```suggestion
on:
  pull_request:
    types: [ opened, synchronize ]
  push:
```